test/integration-serial/resource_discovery_test.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+class ResourceDiscoveryTest < KubernetesDeploy::IntegrationTest
+  def test_non_prunable_crd
+    assert_deploy_success(deploy_fixtures("resource-discovery", subset: ["shredders.yml"]))
+    assert_deploy_success(deploy_fixtures("resource-discovery", subset: ["shredders_cr.yml"]))
+    # Deploy any other non-priority (predeployable) resource to trigger pruning
+    assert_deploy_success(deploy_fixtures("hello-cloud", subset: ["configmap-data.yml"]))
```

Comment:
let's add a configmap (or whatever) to the fixture set we're using instead of deploying an unrelated one. The fixture sets are intended to be representing apps, so deploying a different one to the same ns is semantically odd.